src/routes/api/playlists/user/[playlistId]/+server.ts

```diff
@@ -9,14 +9,16 @@ export const GET = (async (event) => {
     if (!playlist) {
         throw error(404, `No playlist exists with refId ${playlistId}`);
     }
-    return json(playlist, { headers: [ ['Access-Control-Allow-Origin', "*"] ]});
+    const modifiedPlaylist = { ...playlist, words: playlist.words.map(word => word.map(letters => (letters === false ? null : letters)))}
+    return json(modifiedPlaylist, { headers: [ ['Access-Control-Allow-Origin', "*"] ]});
 }) satisfies RequestHandler;
 
 export const POST = (async (event) => {
     const { playlistId } = event.params;
     const { uid } = await authenticate(event);
     const playlistData = await event.request.json();
-    await writePath(`/playlists/user/${uid}/${playlistId}`, playlistData);
+    const modifiedData = { ...playlistData, words: playlistData.words.map(word => word.map(letters => (letters === null ? false : letters)))};
```

Comment:
Nullish coalescing makes this slightly more compact. 
```suggestion
    const modifiedData = { ...playlistData, words: playlistData.words.map(word => word.map(letter => (letter ?? false)))};
```

Reply:
Also, should we just move this whole function up a level now, so that the POST request just goes to `/playlists/user` or even just `/playlists`? (I forget which one I did for decks, but just matching that)

Reply:
yep can move this up one to match what was done in shared decks! however, i think we discussed also generating the ID in the api and i don't think we're doing that anywhere else yet so for now gonna keep it in godot unless we want to change that everywhere

Reply:
oh also i for some reason thought maybe we would eventually have preloaded playlists, which is why i have user as a subresource but if we think that's never gonna happen i can get rid of it

Reply:
i think summer was thinking of making some preloaded ones though

Reply:
> yep can move this up one to match what was done in shared decks! however, i think we discussed also generating the ID in the api and i don't think we're doing that anywhere else yet so for now gonna keep it in godot unless we want to change that everywhere

Okay I don't know what I was smoking, it seems like in the `decks` I am doing the exact same thing where you POST to `/deck/deckId`. I don't think that is the right way to do it but since we don't currently have a standard way to generate the IDs and we don't wanna have to refactor all of it, let's just keep it like this in both cases for now

Reply:
The preloaded thing is not really a concern because they don't need an API. But let's revisit later

Reply:
Still think you should change to nullish coalescing but then we are good here

Reply:
oh shit lmao i just moved it up to playlists/users but can undo that change